### PR TITLE
Flip defaults to optimize for streaming search by default.

### PR DESCRIPTION
**Pull request by baldersheim**
@vekterli @geirst PR
@toregge @jobergum @kkraune FYI

### CODE REVIEW

File: searchcore/src/vespa/searchcore/config/proton.def [hunk 1]
```diff
@@ -213,20 +213,22 @@ summary.cache.initialentries long default=0 restart
 summary.cache.compression.type enum {NONE, LZ4, ZSTD} default=LZ4 restart
 
 ## Control compression level of the summary while in cache.
+## LZ4 has normal range 1..9 while ZSTD has range 1..19
 summary.cache.compression.level int default=9 restart
```
Comment (vekterli) [inline searchcore/src/vespa/searchcore/config/proton.def]: Is 9 a good default for ZSTD now that we've switched to it as default? Re: comment above with different ranges of values for LZ4 and ZSTD.

**Review by vekterli — APPROVED** [review]
LGTM. Quick question on compression level defaults.

File: searchcore/src/vespa/searchcore/config/proton.def (hunk 1 above)
Comment (baldersheim) [inline searchcore/src/vespa/searchcore/config/proton.def]: Comment updated

Comment (baldersheim) [inline searchcore/src/vespa/searchcore/config/proton.def]: Have elaborated in .def file comments.

**Pull request merged**